### ISSUE TITLE
508-defect-2: Checkbox should be to the left of the issue on mobile (not on the top)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#53476]
### Point of contact

joshkimux

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

[According to the design on sketch](https://www.sketch.com/s/d2416db4-9a4f-4919-abe4-20ba4bdcfd89/p/C22CAF85-3AE8-4837-9C8F-0C7145740CC1/canvas), the checkbox is aligned to the left of the issue which creates a stronger signifier that the two are connected as a checkbox.

### Current
<img width="405" alt="Screenshot 2023-02-10 at 9 58 14 AM" src="https://user-images.githubusercontent.com/14154792/218123191-c3d72c7c-ed77-43f8-a76f-24da8f776749.png">

### Expected
<img width="386" alt="Screenshot 2023-02-09 at 3 11 47 PM" src="https://user-images.githubusercontent.com/14154792/218123041-ba85e453-33be-4358-abde-9ff2d468a820.png">

### Reproduction steps

Reduce the screenwidth to under 450px, confirm the checkbox bumps up to the top.

### Proposed solution or next steps

As documented above, the checkbox should be aligned to the left of the issue.

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [X] Zoom
- [ ] Semantics
- [ ] Axe-core
- [X] Something else

**Comment by joshkimux** [comment]
Confirmed this has been fixed! Brilliant work 🥳 

<img width="496" alt="Screenshot 2023-02-21 at 1 23 49 PM" src="https://user-images.githubusercontent.com/14154792/220428250-c943811e-25f9-4d3a-a704-72e7ab5232e9.png">
